README.md

```
@@ -9,22 +9,33 @@ See the [features page on our wiki](https://github.com/tfrizzell/harnessnation-p
 
 ## Installation
 
-### <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/Google_Chrome_icon_%28February_2022%29.svg/120px-Google_Chrome_icon_%28February_2022%29.svg.png" height="30" valign="text-bottom" /> &nbsp; Google Chrome
+### <img src="https://upload.wikimedia.org/wikipedia/commons/e/e1/Google_Chrome_icon_%28February_2022%29.svg" height="30" valign="text-bottom" /> &nbsp; Google Chrome
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)